src/database/seeds/1728493930771_add-openstax-member.ts

```diff
@@ -1,36 +1,83 @@
 import type { Kysely } from 'kysely'
+import { DB } from '@/database/types'
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export async function seed(db: Kysely<any>): Promise<void> {
-    await db
+export async function seed(db: Kysely<DB>): Promise<void> {
```

Comment:
@nathanstitt anything else of value you think we should add here in the seed while we're updating it?

Reply:
while I understand why we'd want to, I don't think we can do this here 😭 

the seed files run on all environments, including prod, where we do not want testing accounts with semi-public passwords created.

Instead we have a `tests/configure-test-env.ts` that does roughly this, but could probably use more ❤️ 

Also, we can't edit existing seed or db files because they will not re-run on existing db, instead we should create new files using `npx kysely seed` or `npx kysely migrate`